pkg/webui/components/dropdown/dropdown.styl

```diff
@@ -121,14 +108,8 @@ ul.dropdown
       color: $tc-deep-gray
       text-decoration: none
 
-  .submenu-container
-    bottom: 0px
-    left: 0px
-    position: absolute
-    width: 180%
-    height: 5rem
-    z-index: 9999
```

Comment:
This solution was kind of hacky and error-prone. Also we use `$zi` variables for `z-index`.

Reply:
Yeah, I've talked about this with @pierrephz and we determined that there should not be a gap like that. There's a couple of particularities with how these submenus should work, such as hover behavior, delays, and open/close behavior. We will take a more dedicated look at that later.